doc/Language/objects.pod6

```diff
@@ -449,14 +449,27 @@ will itemize; C<$.a> will be equivalent to C<self.a.item> or C<$(self.a)>.
 
 =begin code
 class A {
+    has Int @.numbers;
     has $.x = (1, 2, 3);
-    method b() { .say for self.x; .say for $.x }
+
+    method show-diff() { .say for self.x; .say for $.x }
+
+    method twice  { self.times: 2 }
+    method thrice { $.times: 3    }
+
+    method times($val = 1) { @!numbers.map(* * $val).list }
 };
-A.new.b; # OUTPUT: «1␤2␤3␤(1 2 3)␤»
+
+my $obj = A.new(numbers => [1, 2, 3]);
+$obj.show-diff;   # OUTPUT: «1␤2␤3␤(1 2 3)␤»
+say $obj.twice;   # OUTPUT: «(2 4 6)␤»
+say $obj.thrice;  # OUTPUT: «(3 6 9)␤»
 =end code
```

Comment:
It's nowhere to be seen here either. You are not using obj.twice: whatever, which is another type of colon syntax. So it would be better to clarify what kind of colon you're talking about, the one in signatures or in method calls, and then actually use it in the examples.

Reply:
Since the sentence makes a reference to `self` (which is only available to a method's body), I thought it provided enough context but you're right about adding some clarification.